static/app/utils/discover/fields.tsx

```diff
@@ -381,7 +381,10 @@ export const AGGREGATIONS = {
     parameters: [
       {
         kind: 'column',
-        columnTypes: ['string', 'duration'],
+        columnTypes: validateDenyListColumns(
+          ['string', 'duration'],
+          ['id', 'issue', 'user.display']
```

Comment:
You can't count_if user.display (I'm guessing since it's coalescing)? Wouldn't that be a common one people want to use `count_if` on? 🤔 

Also does this have a good error message if you aren't allowed?

Reply:
it does, backend already blocks these.
issue and user.display aren't allowed yet cause they're field aliases and require extra work to be supported. Its on the backlog, but showing them in the UI when they aren't supported isn't the best experience so I wanted to hide them
<img width="409" alt="image" src="https://user-images.githubusercontent.com/4205004/125669006-da11febe-9bde-4556-ae1c-c4cf90bc7056.png">